deploy/crds/infra.watch_v1beta1_servicetelemetry_cr.yaml

```diff
@@ -34,6 +34,15 @@ spec:
           persistent:
             storageSelector: {}
             pvcStorageRequest: 20Gi
+
+    logs:
+      loki:
+        enabled: false
+        flavor: 1.extra-small
+        replicationFactor: 1
+        storage:
+          objectStorageSecret: test
```

Comment:
can we please add some more obvious secret, rather than `test`?

Reply:
What do you suggest? You can make suggested edits on the web interface.

Reply:
"test" is a name of secret, that comes with the loki-operator and it's preconfigured to work with minio. I guess I could use the stf-run-ci to replace the name of that secret for something else. Does something like "loki-storage-secret" sound OK?

Reply:
Is that the standard / default value that loki-operator uses? If so, then I think this is fine as-is. If it's trivial to name it anything you want, then loki-storage-secret sounds find to me. Thanks!